File: Desafios em C#/2444 - Volume da TV.cs
using System; 

class URI {

    static void Main(string[] args) { 

        string[] linha = Console.ReadLine().Split(' ');

        int volumeInicial = int.Parse(linha[0]);
        int numModificacoes = int.Parse(linha[1]);
        int totalVolume = volumeInicial;
          
        string[] aux = Console.ReadLine().Split(' ');
                
        for(int i = 0; i < numModificacoes; i++){
            totalVolume += int.Parse(aux[i]);
            if(totalVolume < 0) totalVolume = 0;
            else if(totalVolume > 100) totalVolume = 100;
        }
          
        Console.WriteLine(totalVolume);

    }

}

File: Desafios em JavaScript/2444 - Volume da TV.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var ln = lines.shift().split(" ");

var volumeInicial = parseInt(ln.shift());
var numModificacoes = parseInt(ln.shift());
var totalVolume = volumeInicial;

var ln1 = lines.shift().split(" ");
for(let i = 0; i < numModificacoes; i++){
    totalVolume += parseInt(ln1.shift());
    if(totalVolume < 0) totalVolume = 0;
    else if(totalVolume > 100) totalVolume = 100;
}
    
console.log(`${totalVolume}`);